src/components/browse/singleDatasetView/singleDatasetViewAccordion/singleDatasetViewAccordion.tsx

```diff
@@ -11,17 +11,17 @@ const SingleDatasetViewAccordion = (props: SingleDatasetViewAccordionProps) => {
 
   return (
     <Accordion>
-      <Accordion.Item className="mb-4" eventKey="0">
+      <Accordion.Item className="mb-4 border-0" eventKey="0">
```

Comment:
Box border color should be EFF0F6

Reply:
Which box? The one that says 'Experiment Summary'?

Reply:
Top bar boxes, big boxes and yes experiment, file, study boxes. All boxes should be light grey